docs/source/config-options.md

```diff
@@ -249,6 +249,11 @@ EnterpriseGatewayApp options
     Default: False
     Is YARN Kerberos/SPNEGO Security enabled (True/False).
     (EG_YARN_ENDPOINT_SECURITY_ENABLED env var)
+--EnterpriseGatewayApp.zmq_port_ping_interval=<Int>
+    Default: 30000
+    Specifies the value of ws_ping_interval that is being used for websocket
+    ping pong mechanism in ZMQ Port Handler from notebook server.
+    (NOTEBOOK_ZMQ_PORT_PING_INTERVAL env var) 
```

Comment:
I'd prefer we follow the naming (and units) convention that @esevan introduced in https://github.com/jupyter/nb2kg/pull/29.  Let's call the option `ws_ping_interval` (with the help string indicating seconds) and the env variable `EG_WS_PING_INTERVAL_SECS` (defaulting to 30), then expand to `ms` on usage.

I don't think we need to mention ZMQ since that corresponds more closely to the (multiple) ZMQ ports - we're really pinging the (single) WS port.